Token_Contracts/contracts/StandardToken.sol

```diff
@@ -16,25 +16,23 @@ contract StandardToken is Token {
         //Default assumes totalSupply can't be over max (2^256 - 1).
         //If your token leaves out totalSupply and can issue more tokens as time goes on, you need to check if it doesn't wrap.
         //Replace the if with this one instead.
-        //if (balances[msg.sender] >= _value && balances[_to] + _value > balances[_to]) {
-        if (balances[msg.sender] >= _value) {
-            balances[msg.sender] -= _value;
-            balances[_to] += _value;
-            Transfer(msg.sender, _to, _value);
-            return true;
-        } else { return false; }
+        //require(balances[msg.sender] >= _value && balances[_to] + _value > balances[_to]);
+        require(balances[msg.sender] >= _value);
```

Comment:
Maybe I don't understand what you're asking? I didn't remove anything, I just changed the ifs to requires and removes the else blocks. Both what are inside the `require` statements and what follows them are otherwise the same as what were inside and following the `if` statements.

You're talking about this? `balances[_to] + _value > balances[_to]`? It was commented out in the code originally. `If your token leaves out totalSupply`...`Replace the if with this one instead.`

Should that be the default behavior?